api/libretime_api/settings.py

```diff
@@ -128,7 +141,7 @@
 
 LANGUAGE_CODE = "en-us"
 
-TIME_ZONE = "UTC"
+TIME_ZONE = CONFIG.api.time_zone
```

Comment:
We probably want a way to sync this with the UI config option in the DB at some point

Reply:
We already have a timezone-via-ui setting, so we define it in two places. Also the playout uses the system timezone to determine playout. Not something to fix now, but we do need to find consistency somewhere

Reply:
Yes, we should have this timezone setting in a single place. For that reason moving this configuration entry to the configuration file would make this a lot easier to share across services.

This topic is actually part of #1451